hs_core/hydroshare/utils.py

```diff
@@ -540,19 +563,19 @@ def prepare_resource_default_metadata(resource, metadata, res_title):
 
     if add_rights:
         # add the default rights/license element
-        metadata.append({'rights':
-                             {'statement': 'This resource is shared under the Creative Commons Attribution CC BY.',
-                              'url': 'http://creativecommons.org/licenses/by/4.0/'
-                             }
-                        })
+        statement = 'This resource is shared under the Creative Commons Attribution CC BY.'
+        url = 'http://creativecommons.org/licenses/by/4.0/'
+        metadata.append({'rights': {'statement': statement, 'url': url}})
 
     metadata.append({'identifier': {'name': 'hydroShareIdentifier',
-                                    'url': '{0}/resource{1}{2}'.format(current_site_url(), '/', resource.short_id)}})
+                                    'url': '{0}/resource{1}{2}'.format(current_site_url(), '/',
+                                                                       resource.short_id)}})
```

Comment:
@pkdash I think it is better to include the ```/``` directly within the string rather than replacing ```{1}``` for the ```url``` string